README.md

```diff
@@ -1,88 +1,127 @@
-## Travel Tracker
+# Travel Tracker
 
-### Overview
+> Travel Tracker is a web application which manages and tracks different trips for users and travel agents, by consuming and posting to an external API.
 
-Travel Tracker is an application which manages and tracks different trips for
-users and travel agents, by consuming and posting to an external API.
-This is a project made by [Joshua Everett](https://github.com/josh-everett01).
-This project implements HTML / CSS for the main skeleton design and styling, and
-uses JavaScript to authorize user login, make calls to the API, and to populate
-travelers / trips information on the user dashboard.
+[Travel Tracker Live Link](https://josh-everett01.github.io/travel_tracker/)
+This project was created by [Joshua Everett](https://github.com/josh-everett01)
 
-#### Login
+## Table of contents
 
-**Upon opening / navigating to the project:**
-The user sees a login screen which
-accepts a username based on either a traveler or an agent of a travel agency.
-When a traveler logs in a fetch call is made to the API to verify the travelers
-ID against the ID's in the Array of travelers.
+- [Overview](#overview)
+- [Screenshot](#screenshot)
+- [Setup](#setup)
+- [Tech Stack](#tech-stack)
+- [Learning Objectives](#learning-objectives)
+- [Credits](#credits)
+- [Contact](#contact)
 
-#### Traveler Dashboard
+# Overview
 
-**If a traveler logs in they see a dashboard page showing them:**
+Travel Tracker utilizes the fetch API to gather data from multiple endpoints. A user is able to login as either an agent or traveler, and then they are able to view trips, make trip requests, and approve / delete trip requests depending on who is logged in.
 
-- All trips taken (past, present, upcoming, and pending)
-- Total amount of money spent on trips this year (This
-  is calculated from the trips data and includes a 10%
-  agent fee)
+#### To login as an agent use the following login information:
 
-#### Agent Dashboard
+    Username: agency
+    Password: travel2020
 
-**If an agent logs in they see a dashboard page showing them:**
+####As an agent who is logged in you will:
 
-- New trip requests (a users pending trips)
-- Total income generated this year (will be 10% of user
-  trip cost)
-- Travelers on trips for today's date
+- See any pending trips
+- Be able to approve any pending trips
+- See any trips happening on today's date
+- Be able to search for a traveler and then approve or delete any trips for that traveler
 
-#### Traveler Interaction
+#####To approve a pending trip:
+Locate the trip you would like to approve in the PENDING TRIPS section
```

Comment:
why not make this a bulleted list?  would probably look nicer

look up markdown cheat sheet for more reference on how to make this document look pretty